server/tserver/src/main/java/org/apache/accumulo/tserver/scan/ScanTask.java

```diff
@@ -138,6 +175,13 @@ public T get(long timeout, TimeUnit unit)
     return rAsT;
   }
 
+  @Override
+  public T get(long timeout, TimeUnit unit)
+      throws InterruptedException, ExecutionException, TimeoutException {
+    // TODO probably no longer makes sense to extend future
```

Comment:
I think we can satisfy this TODO by creating a separate issue for this change. ScanTask implemented Future<> before this PR.

Reply:
This one does go with this PR.  The changes to add busy timeout changed the class such that it no longer makes sense to extend future.